src/plugins/blocksmith/components/image-block-modal/CustomizeOptions.jsx

```diff
@@ -52,113 +54,131 @@ const CustomizeOptions = ({
   selectedImage,
   updatedDate,
   url,
-}) => (
-  <div className="modal-body-blocksmith">
-    <ImageBlockPreview className="my-4" block={block} />
-    {hasCaption && (block.has('caption') || selectedImage.has('description')) && (
-      <p>Caption: {block.get('caption') || selectedImage.get('description')}</p>
-    )}
-    <FormGroup>
-      <ImageAssetPicker
-        multiAssetErrorMessage="Invalid Action: Trying to assign multiple Gallery Block Poster images."
-        isImageSelected={!!selectedImage}
-        isModalOpen={isAssetPickerModalOpen}
-        isDisabled={false}
-        label="Select an image"
-        node={node}
-        onClearImage={handleClearImage}
-        onSelectImage={handleSelectImage}
-        onToggleAssetPickerModal={handleToggleAssetPickerModal}
-      />
-    </FormGroup>
-    <div style={{ maxWidth: '350px', margin: '0 auto' }}>
+}) => {
+  const hasThemeableMixin = memoize(() => node.schema().hasMixin('triniti:common:mixin:themeable'));
```

Comment:
@jalopez13 memoizing this is not needed, use the same strategy we use elsewhere (just check for the mixin)